runtime/go.mod

```diff
@@ -13,8 +13,10 @@ require (
 	github.com/fluxcd/cli-utils v0.36.0-flux.11
 	github.com/fluxcd/pkg/apis/acl v0.5.0
 	github.com/fluxcd/pkg/apis/event v0.13.0
+	github.com/fluxcd/pkg/apis/kustomize v1.8.1-0.20250123112748-c55030369b58
```

Comment:
Need to import the proper tag after https://github.com/fluxcd/pkg/pull/857 is merged.